App/HejDTO/Sources/HejDTO/Contests/GET/ContestWinner/ContestsWinners.Embedded.swift

```diff
@@ -0,0 +1,43 @@
+
+import Foundation
+
+extension Contests {
+    
+    ///https://docs.hejto.pl/#tag/Contests/operation/get_contest_winners
+    public struct ContestWinner: Codable, Equatable {
+        public let user: ContestUserWinner
+        public let place: Int
+        public let uuid: String
+        
+        public struct ContestUserWinner: Codable, Equatable {
+            public let username, sex, description, city: String
+            public let avatar, background: Common.Image
+            public let status: String
+            public let controversial: Bool
+            public let currentRank, currentColor: String
+            public let verified, sponsor: Bool
+            public let createdAt: String
+            
+            public struct Links: Codable, Equatable {
+                
+                public let selfLink: Common.Link
+                public let follows: Common.Link
+                
+                private enum CodingKeys: String, CodingKey {
+                    case selfLink = "self", follows
+                }
+            }
+            
+            public let links: Links
```

Comment:
Could you do the same for all types/extensions you wrote? :) You can check this file for reference: https://github.com/sloik/HejtoBuzz/blob/main/App/HejDTO/Sources/HejDTO/Posts/GET/Details/Posts.Details.swift#L78